EngineeringModel.Tests/Dialogs/Rows/ParameterRowViewModelTestFixture.cs

```diff
@@ -148,14 +159,18 @@ public void Setup()
             };
 
             this.cptParameter.ValueSet.Add(this.GetNewParameterValueSet(this.option1, this.stateList.ActualState.First()));
+            this.cptParameter.ValueSet.Add(this.GetNewParameterValueSet(this.option1, this.stateList.ActualState.Skip(1).First()));
```

Comment:
Line 103-105 define the actual states you want here: ```this.actualState1, this.actualState2 and this.actualState3```.

I know that ```First()``` and ```Last()``` were already used here instead of those fields, but ```Skip(1).First()```makes it a bit ugly.
I would suggest to use the ```this.actualState*``` fields directly here.
Of course for all 6 ```this.cptParameter.ValueSet.Add``` calls.